### PR TITLE
Feature/iesp 125

### PR DESCRIPTION
**Pull request by AlessandroColucci**
Fixed query for polling service

### CODE REVIEW

File: src/main/java/it/pagopa/interop/probing/eservice/operations/service/impl/EserviceServiceImpl.java [hunk 1]
```diff
@@ -48,7 +45,7 @@
 import it.pagopa.interop.probing.eservice.operations.util.OffsetLimitPageable;
 import it.pagopa.interop.probing.eservice.operations.util.constant.ErrorMessages;
 import it.pagopa.interop.probing.eservice.operations.util.constant.ProjectConstants;
-import lombok.extern.slf4j.Slf4j;
+import it.pagopa.interop.probing.eservice.operations.util.logging.Logger;
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/eservice/operations/service/impl/EserviceServiceImpl.java:48]: Good.

File: src/main/java/it/pagopa/interop/probing/eservice/operations/service/impl/EserviceServiceImpl.java [hunk 2]
```diff
@@ -128,7 +125,8 @@ public void updateEserviceFrequency(UpdateEserviceFrequencyDto inputData)
   public SearchEserviceResponse searchEservices(Integer limit, Integer offset, String eserviceName,
       String producerName, Integer versionNumber, List<EserviceMonitorState> state) {
 
-    logger.logMessageSearchEservice(limit, offset, eserviceName, producerName, versionNumber, state);
+    logger.logMessageSearchEservice(limit, offset, eserviceName, producerName, versionNumber,
+        state);
     Page<EserviceView> eserviceList;
     List<String> stateBE = Objects.isNull(state) || state.isEmpty() ? List.of()
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/eservice/operations/service/impl/EserviceServiceImpl.java:131]: But state can be null at this step?

**Review by nttdata-rtorsoli — APPROVED** [review]
See comments

**Review by galales — APPROVED** [review]

File: src/main/java/it/pagopa/interop/probing/eservice/operations/service/impl/EserviceServiceImpl.java (hunk 2 above)
Comment (AlessandroColucci) [reply src/main/java/it/pagopa/interop/probing/eservice/operations/service/impl/EserviceServiceImpl.java:131]: Ok, it will be fixed in the IESP-121.

**Pull request merged**